weasyl/controllers/well_known.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from pyramid.response import Response
+
+from libweasyl.exceptions import WeasylError
+from weasyl import define, profile
+
+
+def webfinger(request):
+    """Handle WebFinger requests."""
+    resource = request.params.get('resource')
+    if not resource or not resource.startswith('acct:'):
+        raise WeasylError("Invalid resource parameter")
```

Comment:
Sorry for taking so long to start to take a look at this. The string parameter that the constructor for `WeasylError` takes should match one of the keys of `weasyl.errorcode.error_messages`. The value for that key becomes the error message that is shown to the user when `WeasylError` is thrown.